sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -903,6 +903,16 @@ public String getEntityPath() {
             });
     }
 
+    /**
+     * Package-private method that releases a message.
```

Comment:
Pardon my Java ignorance, but how do we mark something as package private? Are there any particular annotations to be used? Or the lack of one makes it package private?

Nit: Since this is a method to update disposition just like the 4 settlement methods, perhaps a better position for this method is right after those 4 methods?

Reply:
The absence of visibility modifier (private, protected public) in the method signature mark it as package-private, so they are not a part of public API